app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +20,7 @@
         </activity>
 
         <activity
-            android:name="org.researchstack.backbone.ui.ViewTaskActivity"
+            android:name="org.researchstack.backbone.interop.ViewBackboneInteropTaskActivity"
             android:windowSoftInputMode="adjustResize"
             android:theme="@style/Theme.BackboneApp.Survey"
```

Comment:
Although `:backbone_interop` will merge ViewBackboneInteropTaskActivity into your app's `AndroidManifest.xml`, you may need to update your app's `AndroidManifest.xml` if you specified a theme or other custom attribute.